log_parse.py

```diff
@@ -23,26 +23,24 @@ def handle_message(self, df, id):
         elif id == 114001:
             # %ASA-1-114001: Failed to initialize 4GE SSM I/O card (error error_string).
             m = re.search(r'card \(error (\w+)\)', df.loc[id, 'Text'])
-            if m:
-                df.loc[id, 'Error'] = m.group(1)
+        elif id == 114009:
```

Comment:
Here is an example of where it looks like you have merged your branch incorrectly with upstream.  I'm not entirely sure how this happened.  If you look at [this](https://github.com/NCATComp410/comp410_summer21/blob/4ab66d1e8c16adf13a5e2dd5394e33875f08e665/log_parse.py#L23) line of code, you'll see how it should look.  You shouldn't be removing the if m: clause and the following line